### PR TITLE
Fixes performance issues on Linux, Run @60fps

### LINKED ISSUE #309: Framerate locked at 32 fps on Linux
For some reason Bespoke runs extremely poor for me on X11/Openbox with or without compositor on intel igpu. Fps is locked at 32

### PR DESCRIPTION
**Pull request by pushqrdx**
This should fix #309 which originated from #30. Original issue seems to be caused by how JUCE sends batch render calls to the gpu causing hangs with mesa/dri leading to dropped input events, after fiddling around a bit i found that if you're handling rendering yourself (not using the paint override) like Bespoke does, you should set `ComponentPaintingEnabled` to false to improve performance and avoid redundant call to paint. 
Note: this might also benefit performance on other platforms.

From Juce Docs:
> By default this is set to true. If you're not using any paint() method functionality and are doing all your rendering in an OpenGLRenderer, you should disable it to improve performance.

---

This fixes the issue on my machine and Bespoke runs at 60fps with no issues.

### CODE REVIEW

**Comment by cassaundra** [comment]
I'm glad to see this issue being address! Bespoke feels much smoother when at 60 fps. Unfortunately, I still seem to be getting the same input issues as before, on X11/Mesa.

File: Source/MainComponent.cpp [hunk 1]
```diff
@@ -121,11 +122,7 @@ class MainContentComponent   : public OpenGLAppComponent,
       
       mSynth.Poll();
       
-#if DEBUG || BESPOKE_LINUX
```
Comment (awwbees) [inline Source/MainComponent.cpp:124]: could you leave the DEBUG check in, and just remove BESPOKE_LINUX here? I still want to run at 30fps in debug mode

**Comment by cassaundra** [comment]
Aha, I was wrong: it turns out this fix does work. After an hour of troubleshooting and digging through JUCE internals, it turns out I was just executing an old binary. It seems the release directory was moved from `./ignore/build/BespokeSynth_artefacts/Release` to `./ignore/build/Source/BespokeSynth_artefacts/Release` in 1f68e48b765430d65b46e21b93cd22d4eef540e0, although I'm not sure if this was an intentional side effect.

File: Source/MainComponent.cpp (hunk 1 above)
Comment (pushqrdx) [reply Source/MainComponent.cpp:124]: @awwbees sorry forgot to add it back :D

**Comment by awwbees** [comment]
amazing!! thank you!!!!

**Comment by awwbees** [comment]
I unfortunately had to temporarily undo this change, it is leading to thread safety crashes. it seems that I have some thread-unsafe code in my rendering, and the fix in this PR uncovered these issues. I need to fix these issues, so I can restore the fix in this PR, but until I do that, I need to undo this PR

if you still wish to run with this change (and risk random crashes), you can re-enable the behavior by generating your cmake project with `-DBESPOKE_LINUX_HIGH_FPS_WITH_THREAD_PROBLEMS:BOOL=TRUE`

**Pull request merged**